packages/button/src/test/mwc-button.test.ts

```diff
@@ -22,51 +22,89 @@ import {fixture, rafPromise, TestFixture} from '../../../../test/src/util/helper
 
 const ICON_SELECTOR = '.mdc-button__icon';
 
-const noText = html`<mwc-button></mwc-button>`;
-
-const basic = html`
-  <mwc-button>this is a button</mwc-button>
-`;
+interface ButtonProps {
+  disabled: boolean;
+  unelevated: boolean;
+  raised: boolean;
+  dense: boolean;
+  outlined: boolean;
+  text: string;
+  label: string;
+  icon: string;
+  trailingIcon: boolean;
+}
+
+const button = (propsInit: Partial<ButtonProps> = {}) => {
+  return html`
+    <mwc-button
+      ?disabled=${propsInit.disabled}
+      ?unelevated=${propsInit.unelevated}
+      ?raised=${propsInit.raised}
+      ?dense=${propsInit.dense}
+      ?outlined=${propsInit.outlined}
+      ?trailingIcon=${propsInit.trailingIcon}
+      label=${propsInit.label ?? ''}
+      icon=${propsInit.icon ?? ''}>${propsInit.text ?? ''}</mwc-button>
+  `;
+};
```

Comment:
@e111077 i tried it here fyi, if you want a read. and each property now has a suite, with the right setup.